src/cascadia/TerminalApp/Pane.cpp

```diff
@@ -612,7 +612,13 @@ void Pane::UpdateSettings(const TerminalSettings& settings, const GUID& profile)
     {
         if (profile == _profile)
         {
-            _control.UpdateSettings(settings);
+            // Update the parent of the control's settings object (and not the object itself) so
+            // that any overrides made by the control don't get affected by the reload
+            auto child = winrt::get_self<winrt::TerminalApp::implementation::TerminalSettings>(_control.Settings());
+            auto parent = winrt::get_self<winrt::TerminalApp::implementation::TerminalSettings>(settings);
+            child->ClearParents();
+            child->InsertParent(0, parent->get_strong());

```

Comment:
isn't 
```suggestion
            child->InsertParent(0, *parent);
```
usually the more idiomatic way for turning a `com_ptr<impl::Foo>` into a projected `Foo`?

Reply:
We want a `com_ptr` from the `Foo` in this case though!

Reply:
oh derp good point